### PR TITLE
Invisible icon fix

### PR DESCRIPTION
**Pull request by UndeadNappist**
Some cleanup and fixes, adding icon states, adding the ability to craft soap in the wild, etc.

### CODE REVIEW

File: code/modules/clothing/head/collectable.dm [hunk 1]
```diff
@@ -4,6 +4,7 @@
 /obj/item/clothing/head/collectable
 	name = "collectable hat"
 	desc = "A rare collectable hat."
+	icon_state = "viking"
```
Comment (LetterN) [inline code/modules/clothing/head/collectable.dm]: this should fallback into the default "no-texture" texture

File: lewd/objects.dm [hunk 2]
```diff
@@ -73,7 +73,8 @@
 	//var/blood_DNA = list()
 	beauty = -50
 
-/obj/effect/decal/cleanable/cum/New()
+/obj/effect/decal/cleanable/cum/Initialize()
 	..()
 	dir = pick(1,2,4,8)
+	reagents.add_reagent("cum", rand(8.13))
```
Comment (LetterN) [inline lewd/objects.dm]: >'.' instead of ,

Comment (LetterN) [inline lewd/objects.dm]: rand(8,13)

File: code/modules/clothing/head/collectable.dm (hunk 1 above)
Comment (UndeadNappist) [reply code/modules/clothing/head/collectable.dm]: It spawns on raiders sometimes as an invisible hat, and it's annoying. Either this or removing it from the spawn chance.

**Comment by LetterN** [comment]
Just fix the conflicts and it's good to go!

**Pull request merged**